ocs_ci/utility/utils.py

```diff
@@ -2356,16 +2358,27 @@ def run_ceph_health_cmd(namespace):
 
     """
     # Import here to avoid circular loop
-    from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+    if namespace == constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE:
+        from ocs_ci.utility.connection import Connection
+        from ocs_ci.deployment.helpers.external_cluster_helpers import (
+            get_external_cluster_client,
+        )
 
-    try:
-        ct_pod = get_ceph_tools_pod(namespace=namespace)
-    except (AssertionError, CephToolBoxNotFoundException) as ex:
-        raise CommandFailed(ex)
+        host, user, password, ssh_key = get_external_cluster_client()
+        connection_to_cephcluster = Connection(host=host, user=user, password=password)
+        return connection_to_cephcluster.exec_cmd("ceph health")
```

Comment:
If possible , I prefer to have toolbox pod for external mode as well.
This function is suppose to return only string, but you are returning tuple in this case, and you are checking only the health of external cluster here. I think we should do check on ceph health of both Internal and External cluster.

Reply:
No toolbox in multi-storagecluster. Check my comment above.

Reply:
The function now return only strings and no need to check if tuple.